sdk/java-sdk/src/main/scala/kalix/javasdk/impl/AnySupport.scala

```diff
@@ -489,4 +489,5 @@ private[kalix] object ByteStringEncoding {
 trait MessageCodec {
   def decodeMessage(any: ScalaPbAny): Any
   def encodeScala(value: Any): ScalaPbAny
+  def encodeJava(value: Any): JavaPbAny
```

Comment:
in same situations, we do need JavaPbAny see my obsolete comment here https://github.com/lightbend/kalix-jvm-sdk/pull/1012#discussion_r893635648

It turns out that it started to failed when passing ScalaPbAny. I didn't want to investigate it further. Probably something we could fix, but ok to also have an `encodeJava` for now.